Unplugged tab on homepage and books linking to pdfs

danipop1 opened the pull request:
Created a stand-alone unplugged button on the homepage, to be separate from the backend schema book category tabs. Altered click functionality to send you to a pdf if book is marked as Unplugged. The pdfs are stored in the public folder, as they cant be in minio and work properly

noahpro99 commented:
PDF should probably go in the hosted bucket instead of the git history

noahpro99 commented:
gotcha

On Tue, Nov 12, 2024 at 4:01 PM danipop1 ***@***.***> wrote:

> are you talking about the minIO? I tried that but it doesn't seem to work
> with pdfs the way it does with pngs or jpegs. I mentioned this to dr
> hamouda and she said the current way should be fine
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/codekids-vt/codekids/pull/386#issuecomment-2471572699>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFPHENVCMASG6ZGN74RCOM32AJUB7AVCNFSM6AAAAABRR335ROVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDINZRGU3TENRZHE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

scout517 commented:
The PDF for the maze solver game isn't good. The images on the last pages are too big. I fixed this in the Word file so you can export the pdf again.

Also, can you make me a reviewer?

danipop1 commented:
> The PDF for the maze solver game isn't good. The images on the last pages are too big. I fixed this in the Word file so you can export the pdf again.
> 
> Also, can you make me a reviewer?

I updated the pdfs, and added the card game pptx as a pdf for an additional book as well. You should also be a reviewer now.

scout517 commented:
FYI, you are not supposed to manually edit any files in the frontend API folder. They are automatically generated by Prisma from the backend. I didn't know this either until a couple of weeks ago.

I committed a couple of things since this branch was out of sync with the current backend. I added back UNPLUGGED as a book category since that's what Prisma has and filtered out that category in HomePage.tsx since that button is being created manually.

I also moved the unplugged books to their own file to save space in HomePage.tsx.
